### DISCUSSION TITLE
Relationships between data frames

### DISCUSSION THREAD

**Discussion by gustavdelius** [NOAA-EDAB/ms-keyrun#77] [Q&A] [answered]
The data frame `simSurveyAgeLencomp` contains the number of fish as a function of both age class and length bin. There are also data frames `simSurveyLencomp` and `simSurveyAgecomp`. Am I correct in assuming that they are simply obtained by summing over year classes or length bins respectively in `simSurveyAgeLencomp` . Or am I missing some subtlety and there is additional information to be found in `simSurveyLencomp` and `simSurveyAgecomp`?

**Comment by sgaichas** [comment, marked as answer]
@gustavdelius this is mostly correct; the `simSurveyLenComp` is summed over all age classes in `simSurveyAgeLenComp`. However, there is a subtle difference because `simSurveyAgeComp` is *annual* ages rather than age classes as in `simSurveyAgeLenComp`. 

We started with just the length comp and the annual age comp, but I needed the interim product with length by age class to map lengths into the simulated diet data, which is by age class.

I hope this helps.
